webapp/app/compare.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html ng-app="perf">
+<head>
+    <meta charset="iso-8859-1" />
+    <title>Perfherder Graphs</title>
+    <!-- build:css css/perf.min.css -->
+    <link href="css/bootstrap.css" rel="stylesheet" media="screen">
+    <link href="css/perf.css" rel="stylesheet" media="screen">
+    <!-- endbuild -->
+    <link id="favicon" type="image/png" rel="shortcut icon" href="img/tree_open.png">
+    <style>
+    </style>
+</head>
+<body>
+
+  <section ui-view>
+  </section>
+
+  <script src="js/config/local.conf.js"></script>
+
+  <!-- build:js js/perf.min.js -->
```

Comment:
This will override the perf.js file. Let's change perf.min.js -> compare.min.js. As @edmorley pointed out, you'll also need to make some gruntfile changes for the minification to work in production.